kubernetes/che-plugin-registry/values.yaml

```diff
@@ -13,3 +13,5 @@ chePluginRegistryMemoryLimit: 256Mi
 
 global:
   ingressDomain: 192.168.99.100.nip.io
```

Comment:
@monaka Do you think it would be better to move it under `ingress` field. like
```yaml
global:
  ingress:
    domain: 192.168.99.100.nip.io
    class: 'nginx'
```
Or it would be better to leave as is for backward compatibility?

Reply:
If I had to choice... it's better to rename `ingressDomain` in the another PR.
But I'm not picky about it.